shared/src/commonMain/kotlin/co/nimblehq/blisskmmic/presentation/model/AccountUiModel.kt

```diff
@@ -0,0 +1,7 @@
+package co.nimblehq.blisskmmic.presentation.model
+
+data class AccountUiModel (
+    val imageUrl: String?,
```

Comment:
@blyscuit Do you think  `avatarUrl` is better than `imageUrl`? 🤔

Reply:
Good catch, fixed.